setup.py

```diff
@@ -112,6 +112,7 @@ def cythonize(x, compile_time_env=None):
                          "src/hafnian_approx.hpp",
                          "src/torontonian.hpp",
                          "src/permanent.hpp",
+                         "src/hermite_multidimensional.hpp",

```

Comment:
how come this file is called `hermite_multidimensional`, and not `batch_hafnian`?

Reply:
Recent changes in naming of files.